rasa/cli/x.py

```diff
@@ -351,6 +353,17 @@ def rasa_x(args: argparse.Namespace) -> None:
 
     _configure_logging(args)
 
+    if version.parse(rasa.version.__version__) >= version.parse(
+        MINIMUM_COMPATIBLE_VERSION
+    ):
+        rasa.shared.utils.io.raise_warning(
+            f"Your version of rasa '{rasa.version.__version__}' is currently "
```

Comment:
How do we make sure we remember to fix this once we have Rasa X support? 🤔

Reply:
I could just open a wrap-up issue for the future and have it in our inbox already, potentially also ask Ty what labels he wants to use for the rasa-x support issues.